Tests/TestAppUITests/TestAppUITests.swift

```diff
@@ -84,18 +89,32 @@ class MissingScreen: ScreenObject {
     }
 }
 
-/// A screen that doesn't exist with multiple required elements. Use it to test the init failure
+/// A screen where the first element does not exist. Use it to test the init failure
 /// behavior.
-class MissingMultipleElementsScreen: ScreenObject {
+class MissingFirstElementScreen: ScreenObject {
+
+    init(app: XCUIApplication) throws {
+        try super.init(
+            expectedElementGetters: [
+                { $0.staticTexts["this screen does not exist"] },
+                { $0.staticTexts["Hello, world!"] }
+            ],
+            app: app
+        )
+    }
+}
+
+/// A screen where the first element exists, but the second does not. Use it to test the init passing,
+/// and `isLoaded` failure behavior.
+class MissingSecondElementScreen: ScreenObject {
 
     init(app: XCUIApplication) throws {
         try super.init(
             expectedElementGetters: [
                 { $0.staticTexts["Hello, world!"] },
                 { $0.staticTexts["this screen does not exist"] }
             ],
-            app: app,
-            waitTimeout: 1 // We know the screen is not there, let's not wait for it for long
```

Comment:
I removed the custom timeout because it appears that the screen was expectedly failing the `testScreenInitThrowsWhenScreenWithMultipleElementsIsNotLoaded ()` test not because of really missing 2nd element, but actually because of missing first element, which exists. 1-second delay between the app launch screen instantiation was too short, so I just went for default timeout.